src/WinGetUtil/Exports.cpp

```diff
@@ -247,6 +255,106 @@ extern "C"
     }
     CATCH_RETURN()
 
+    DEFINE_ENUM_FLAG_OPERATORS(WinGetValidateManifestResult);
+
+    WINGET_UTIL_API WinGetValidateManifestV3(

```

Comment:
Add result of discussion `WinGetUtil.h`...

Should we make this a little more complicated from the number of APIs, but less complicated for each?

Imagine:
```
WinGetCreateManifest(path, message, merge path, option, manifest handle);
WinGetValidateManifestV3(manifest handle, index, result, message, operation, option);
WinGetCloseManifest(manifest handle);
```

The create is basically V2 validate, but we return a pointer to reuse for future validations.  It performs the direct manifest validations only.  The returned pointer saves continually parsing the manifest and enables efficient splitting of the various validations we perform.  Then the V3 validate can be "perform this one validation".